Chatkit/Redux/State/UserListState.swift

```diff
@@ -0,0 +1,14 @@
+
+struct UserListState: State {
+    
+    // MARK: - Properties
+    
+    let users: [String : UserState]
+    
+    static let empty: UserListState = UserListState(users: [:])
+    
```

Comment:
Any merit in adding a convience constructor that takes an **array** of users and maps the keys from the user identifier?

Reply:
Same applies to `RoomListState`

Reply:
This is a nice to have, so let's add this in future.